katas/content/nonlocal_games/chsh_classical_strategy/Verification.qs

```diff
@@ -0,0 +1,25 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Random;
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        mutable wins = 0;
+        for i in 1..1000 {
+            let x = DrawRandomInt(0, 1) == 1 ? true | false;
```

Comment:
We have DrawRandomBool now.

Reply:
I will update this line in next (CHSH quantum) PR